backend/src/utils/route-security.ts

```diff
@@ -261,6 +261,12 @@ export const sanitizeNotebookForSecurity = async <
     if (container.name === 'oauth-proxy') {
       return;
     }
+
+    if (container.envFrom) {
+      fastify.log.warn(`${username} submitted a Notebook that contained an envFrom`);
+      delete container.envFrom;
```

Comment:
but envFrom will be allowed at some point if done via ODH Dashboard GUI, right? I mean, there are tons of use cases where an assembled Notebook CR would be cool having env values from secrets or configmaps.

Reply:
Feel free to log a request @shalberd -- as it stands we have no intent to close this gap.

Reply:
So this is about a user who outside of odh-dashboard somehow submitted or applied a Notebook CR?
I am mostly wondering because elsewhere, when I suggested, via ODH Dashboard GUI, being able to add envs from secrets or configmaps, which would result in a container.envFrom, you were open to the idea.

Reply:
regarding even being able to patch existing notebooks ... would one not need access credentials and RBAC for that?

Reply:
I was also worried this would override changes related to [custom CA location](https://github.com/opendatahub-io/kubeflow/pull/43/files#diff-1df80e242be90c94fce3ffb051a0bfe706fe924dab62438c7aa0a186b8067153R283), but that is ok since it only uses env var injection and mounts the trusted-ca configmap into a volume, whose mount path is in a container env var, so that is cool.

Reply:
Perhaps I misspoke... I didn't look clearly at this before I replied. I think I had another scenario in my mind.

`envFrom` is used in the delivery of resources from the Workbench creation screen. We do this now.

The Jupyter Notebooks were moved to the backend and this isn't logic we have anymore. Back when we did this PR -- We had security issues where you could mount your Jupyter notebook against secrets for other users by submitting it in the shared `notebookNamespace`. That was what this logic was after. Since it is on the backend now, the logic is not as worrisome. 

Apologizes for the confusion -- it works as we have previously discussed.